src/fuel_results.jl

```diff
@@ -115,10 +115,9 @@ function make_fuel_dictionary(sys::PSY.System, mapping::Dict{NamedTuple, String}
     for gen in generators
         fuel = hasmethod(PSY.get_fuel, Tuple{typeof(gen)}) ? PSY.get_fuel(gen) : nothing
         category = get_generator_category(fuel, PSY.get_primemover(gen), mapping)
-        push!(gen_categories["$category"], PSY.get_name(gen))
+        push!(gen_categories["$category"], (string(typeof(gen)), (PSY.get_name(gen))))
```

Comment:
Can we just use the the type and not the string interpretation of the type?

Reply:
well the variable name is a symbol (not a datatype) so I think so...